lib/BloqadeQMC/src/ising/Rydberg.jl

```diff
@@ -1,23 +1,23 @@
 using Base.Iterators
 
-
 abstract type AbstractRydberg{O <: AbstractOperatorSampler} <: AbstractLTFIM{O} end
 
-struct Rydberg{O,M <: UpperTriangular{Float64},UΩ <: AbstractVector{Float64}, Uδ <: AbstractVector{Float64}, L <: Lattice} <: AbstractRydberg{O}
+struct Rydberg{O,M <: AbstractMatrix{Float64},UΩ <: AbstractVector{Float64}, Uδ <: AbstractVector{Float64}, A} <: AbstractRydberg{O}
```

Comment:
consider name it more explicitly other than just `Rydberg`

Reply:
Yeah, I'm hoping to rename it to RydbergSampler but again that will be in a different PR